doc/xversionmessage.md

```diff
@@ -84,22 +81,18 @@ than this are illegal.
 ### Later extensions
 
 Implementations should allow extra bytes following the defined fields
-in the `xversion` message as well as any bytes in in the `xverack`
-message, to allow for further extensibility.
+in the `xversion` message to allow for further extensibility.
 
 
 ## Handling and sequencing of `xversion` messages
 
 A node should expect an `xversion` message to arrive after the
-acknowledgement ('verack') of the `version` message.  Only a single
-`xversion` message should arrive during the lifetime of a connection
-and receival of multiple such messages should be considered
-misbehavior of the remote peer.
+`version` message.  Only a single `xversion` message should arrive
+during the lifetime of a connection and receival of multiple such
+messages should be considered misbehavior of the remote peer.
 
 After receival of an `xversion` message, a node must answer with an
-empty `xverack` message to confirm recept. Receival of an `xverack`
-without having ever send an `xversion` message should be handled as a
-protocol violation.
+empty `verack` message to confirm recept.
```

Comment:
why an empty verack...don't we already send a VERACK message...is this a duplicate VERACK or the same one we already send?  and why not still use the XVERACK?

Reply:
verack message is always empty. This is the same one that we already send. i will make that more clear. 

With xversion enabled handshake would go
version xversion verack
without it
version verack

inserting xversion before verack removes the need to xverack.